reading.Rmd

```diff
@@ -1133,7 +1139,7 @@ we should abide by when using the API.
 **Using `rtweet`**
 
 After checking the Twitter website, it seems like asking for 400 tweets one time is acceptable.
-So we can use the `get_timelines` function to ask for the last 400 tweets from the Tidyverse account.
+So we can use the `get_timelines` function to ask for the last 400 tweets from the \@tidyverse account.
```

Comment:
The footnote on this page is in the bottom middle of the page in the pdf rather than at the bottom -- is there a way to correct this?

Reply:
Again a spacing problem.